manager/manifests/fluentd.yaml

```diff
@@ -168,9 +169,6 @@ metadata:
   name: fluentd
   namespace: default
 spec:
-  selector:
```

Comment:
Good catch! Do you mind moving this to a separate PR, along with the change in `operator.yaml`?